sched/adaptdl_sched/controller.py

```diff
@@ -363,6 +366,15 @@ async def _create_pod(self, job_metadata, pod_template,
                 "mountPath": "/dev/shm",
             })
             container.setdefault("env", [])
+            if NETWORK_INTERFACE != "default":
+                container["env"].append({
+                    "name": "NCCL_SOCKET_IFNAME",
+                    "value": NETWORK_INTERFACE
+                })
+                container["env"].append({
+                    "name": "NCCL_DEBUG",
+                    "value": "INFO"
+                })
```

Comment:
Do these configs need to be applied by the controller? Can't the user submitting the job just configure their pod template accordingly?

Reply:
ok, I will move this to NNI side.